logicrunner/builtin/contract/member/member.go

```diff
@@ -316,17 +324,25 @@ func (m *Member) getNodeRef(publicKey string) (interface{}, error) {
 	return nodeRef, nil
 }
 
+type ContractCreateMemberOutput struct {
```

Comment:
ContractCreateMemberOutput -> CreateResponse

Reply:
Maybe like this ContractCreateMemberOutput -> ContractCreateMemberResponse?

Reply:
yes, its ok too